_summaries/tancik2020fourier_1.md

```diff
@@ -1,19 +1,91 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: tancik2020fourier
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ayushchauhan
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## The core idea
+
+The paper proposes that using a Fourier feature mapping of the inputs allows a coordinate-based MLP to learn **high frequency functions** in low dimension problems such as images and 3D objects. The Fourier features are a more general case of the sinusoidal position encoding. It shows using kernel regression (with a Neural Tangent Kernel (NTK)) that standard MLPs without Fourier features are not well suited for coordinate-based tasks such as shape representation, texture synthesis, shape inference, novel view synthesis because of their **spectral bias** towards low frequency functions.
+
+Leveraging literature that approximates deep networks with kernel regression, the paper theoretically explains the reason for this spectral bias and then shows how the Fourier features help in overcoming the problem. Specifically, it proves that the Fourier features transform the NTK into a stationary kernel with tunable frequency components -- thus allowing the network to learn high frequency functions. It also proves the above claims empirically through a few simple experiments.
+
```

Comment:
The summary could be a bit long, especially if you explain the equations. So I would suggest shortening the intro a bit. e.g. this line "coordinate-based tasks such as shape representation, texture synthesis, shape inference, novel view synthesis" you can only give one example.

Reply:
Done.